manifests/addons/dashboards/istio-mesh-dashboard.json

```diff
@@ -1052,7 +1052,7 @@
           "link": false,
           "linkTargetBlank": false,
           "linkTooltip": "Workload dashboard",
-          "linkUrl": "/dashboard/db/istio-workload-dashboard?var-namespace=$__cell_2&var-workload=$__cell_",
+          "linkUrl": "/dashboard/db/istio-workload-dashboard?var-namespace=${__cell_2:raw}&var-workload=${__cell_:raw}",
```

Comment:
can you explain how the raw values of the cell fixes the absolute URL issue? i'm not sure I follow entirely.

Reply:
I was trying to fix that absolute link issue. But we found that is not an issue. 

This PR is just an improvement, which is to avoid appending grafana url before absolute link. (something looks like: `http://localhost:8000/http://localhost:8000/dashboard/snapshot/`)

ref: https://github.com/grafana/grafana/issues/12560

Reply:
OK. Can you update the title/description to reflect the changes?

Reply:
updated!